lib/dom.js

```diff
@@ -3568,7 +3568,35 @@ declare class ClientRectList { // extension
   [index: number]: ClientRect;
 }
 
-// TODO: HTML*Element
```

Comment:
I'm not a maintainer here, but I think this `// TODO` comment should probably be kept - since this PR doesn't actually address it.

Reply:
@motiz88 Thanks your review.
Yeah, I agree with you and I've  pushed revert commit.